testplan/web_ui/testing/src/Nav/NavList.js

```diff
@@ -55,7 +55,7 @@ class NavList extends Component {
       navButtons.push(
         <ListGroupItem
           tabIndex={tabIndex.toString()}
-          key={entry.uid}
+          key={entry.uid+entry.name}
```

Comment:
Why is this needed? Do some entries have the same UID...?

Reply:
actually, they should have unique uids, i though in some cases they might not but it was actually just because of an error i made with the fakeReport for testing, so i will revert this and fix the fake report